build.toml

```diff
@@ -1,6 +1,6 @@
 [archive]
 build_output_dir = 'Built'
-archive_location = '\\nirvana\Measurements\VeriStandAddons\slsc_switch_messaging_library'
+archive_location = '\\nirvana\Measurements\VeriStandAddons\routing_and_faulting_messaging_library'
```

Comment:
I'm not convinced this should be done now, since it's a breaking change for the feed build tool.

Should we be moving to using the repo name directly for these? If we're making a breaking change, some consistency might be nice. 21.0 would be a good opportunity to do a mass-rename of the artifact directories, since we will be rebuilding everything.

Reply:
I'm fine with using the repo name. It's just annoying to have `niveristand` prefixing everything when navigating the archives.

Reply:
Reverting this change for now.